src/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
   "default_locale": "en_US",
-  "version": "16",
+  "version": "17",
```

Comment:
Q: does src/manifest_fenix.json version number need to be in sync, or are they completely separate?

Reply:
The version is not used in fenix. We don't update the extension.